presto-raptor/src/test/java/com/facebook/presto/raptor/metadata/TestDatabaseShardManager.java

```diff
@@ -713,7 +713,7 @@ public void testMaintenanceBlocked()
 
     private long createTable(String name)
     {
-        return dbi.onDemand(MetadataDao.class).insertTable("test", name, false, false, null, 0);
+        return dbi.onDemand(MetadataDao.class).insertTable("test", name, false, true, null, 0, false);
```

Comment:
Why do we change the `organized` argument from `false` to `true`?

Reply:
> Why do we change the `organized` argument from `false` to `true`?

Changed it during self test, will change it back.